data_steward/analytics/cdr_ops/ad_hoc_analyses/CT_RT_clean_ckecks.py

```diff
@@ -0,0 +1,277 @@
+# ---
+# jupyter:
+#   jupytext:
+#     text_representation:
+#       extension: .py
+#       format_name: light
+#       format_version: '1.5'
+#       jupytext_version: 1.7.1
+#   kernelspec:
+#     display_name: Python 3
+#     language: python
+#     name: python3
+# ---
+
+# + tags=["parameters"]
+project_id = ""
+clean_dataset = ""
+sandbox_dataset = ""
+run_as = ""
+# -
+
+# # QC for RT Clean Dataset
+#
+# Quality checks performed on a new RT clean dataset.
+
+# +
+import urllib
+import pandas as pd
+
+from analytics.cdr_ops.notebook_utils import execute
+from utils import auth
+from utils.bq import get_client
+
+pd.options.display.max_rows = 120
+
+# -
+
+SCOPES = [
+    'https://www.googleapis.com/auth/bigquery',
+]
+impersonation_creds = auth.get_impersonation_credentials(run_as, SCOPES)
+
+client = get_client(project_id, credentials=impersonation_creds)
+
+# # Verify measurement records are dropped
+# Set all value_as_number = 9999999 to NULL.  If not null, an error happened.
+
+query = f'''
+SELECT 
+ COUNTIF(value_as_number = 9999999) as n_not_changed,
+ CASE WHEN COUNTIF(value_as_number = 9999999) > 0 THEN 'FAILURE'
+  ELSE 'SUCCESSFUL'
+ END as check_passed
+FROM `{project_id}.{clean_dataset}.measurement`
+'''
```

Comment:
how do you feel about changing these sql statements to using the `JINJA_ENV` formatting?  It is safer than using f-strings to generate the queries.

Reply:
I agree, updated.